src/main.js

```diff
@@ -101,3 +107,10 @@ function checkDuplicate () {
   }
   return false;
 }
+
+function removePoster(target){
+  console.log(savedPosters);
+  savedPosters.splice(target.parentElement.id, 1);
+  target.parentElement.parentElement.removeChild(target.parentElement);
```

Comment:
Refactor these two lines assuming that the `target` parameter is the `article` element (which you want to remove) and can refactor this to the below:

```javascript
savedPosters.splice(target.id, 1);
target.remove();
```